js/functions.js

```diff
@@ -172,6 +172,23 @@ function generateIcons( data ) {
 
          
     // }
+    function fillBarAnimation (){
+        var i,
+        h = window.innerHeight - (elementHeight('#skills')/2),
```

Comment:
Kintamamojo `h` pavadinimas be konteksto labai neaiškus, galbūt pabandyčiau sugalvoti aiškesnį variantą. Nes vėliau skaitant ir pati nesuprasi ką jis turi daryti.